net/switchdev-poller/Makefile

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (C) 2018-2019 Luiz Angelo Daros de Luca
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=switchdev-poller
+PKG_VERSION:=1.0.1
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Luiz Angelo Daros de Luca <luizluca@gmail.com>
+PKG_LICENSE:=GPLv2
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+include $(INCLUDE_DIR)/package.mk
+
+define Package/switchdev-poller
+   SECTION:=net
+   CATEGORY:=Network
+   DEPENDS:=+swconfig
```

Comment:
Shouldn't be depends under TITLE?

Reply:
I wasn't aware that it should be a sequence. There are more pkgs just like that.
Anyway, I'll put it under.